source/Sagan-RDBMS/RDBMSRepositoryMatchingCriteriaBuilder.class.st

```diff
@@ -0,0 +1,47 @@
+Class {
+	#name : #RDBMSRepositoryMatchingCriteriaBuilder,
+	#superclass : #RepositoryMatchingCriteriaBuilder,
+	#category : #'Sagan-RDBMS-Core'
+}
+
+{ #category : #testing }
+RDBMSRepositoryMatchingCriteriaBuilder >> does: anObjectInRepository equal: anObjectInMemory [
+
+	^ anObjectInRepository sequentialNumber notNil AND: [ anObjectInRepository sequentialNumber = anObjectInMemory sequentialNumber ]
+]
+
+{ #category : #testing }
+RDBMSRepositoryMatchingCriteriaBuilder >> does: aStringInRepository includeSubstring: aString [
+
```

Comment:
To avoid unnecesary scares, we should add a comment explaining that `similarTo:` is handled by Glorp